### DISCUSSION TITLE
Failure to to sync up with "Manifest generation error (cached):" "failed timeout after 1m30s"

### DISCUSSION THREAD

**Discussion by noam-codefresh** [argoproj/argo-cd#5303] [Q&A] [answered]
Hello all.
I am playing with argocd, and I am trying to pull some apps from my github repo. I got the applications, but they fail to sync. I keep getting the following errors:
ComparisonError  rpc error: code = Unknown desc = Manifest generation error (cached): `kustomize build /tmp/https:__github.com_<some_repo>/production` failed timeout after 1m30s  2021-01-21 17:32:09 +0200 IST
I tried adding to my argocd-application-controller the following flag in the command:
```
- --repo-server-timeout-seconds
- "180"
```
And also tried adding the following env variables to the argocd-repo-server
`ARGOCD_EXEC_TIMEOUT: "180s"`
So far, nothing seems to work.
I later tried putting in different values to the controller and the repo-server, so I'd have a clue as to which timeout is actually stopping me. I put 240s (4m) in the controller, and 300s (5m) in the repo-server. I see the controllers are re-creating the pods, but I can't seem to get the argo service to retry syncing. Even when I try syncing all my apps manually, I immediately get the "180s" error.
I am guessing that maybe the repo-server is using some cached (failed) value of the manifest, instead of actually generating it again (with the new, updated, timeout values). How can I force it to skip the cache?
Anyone here has any tip or advice for me?

**Comment by jannfis** [comment, marked as answer]
Hey @noam-codefresh , you are right in your assumption that manifest generation errors are cached, at least for the same target revision.

So, to expire the cache, you can use a hard refresh of the application's manifest - either by issuing `argocd app get --hard-refresh <yourapp>`, or by selecting the "Hard refresh" option of the "Refresh" button's drop down in the application view of the UI.

**Reply by noam-codefresh** [reply]
Thanks. I'm currently re-deploying it (I'm playing around with these things at the moment), and I'll give this hard-refresh a try once I encounter the issue again.
